packages/react-search-ui/src/containers/SearchBox.js

```diff
@@ -71,22 +72,27 @@ export class SearchBoxContainer extends Component {
   };
 
   completeSuggestion = searchTerm => {
-    const { setSearchTerm } = this.props;
-    setSearchTerm(searchTerm);
+    const { clearFilters, setSearchTerm } = this.props;
+    setSearchTerm(searchTerm, {
+      ...(clearFilters === false && { clearFilters })
```

Comment:
I was trying to avoid this being passed through:
```
{
  clearFilters: undefined
}
```

I think the only real reason I did that was because it made a bunch of tests fail unintentionally. Probably because they were poorly written tests. I will change this.

Reply:
Ahh interesting. That seems like a scenario which `defaultProps` should be able to handle - or a `clearFilters = true` fallback for non React 🤔

Reply:
For sure. I was thinking I'd do the later.